ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ScanConfigActionsColumn.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { ReactElement } from 'react';
+import { generatePath, useHistory } from 'react-router-dom';
+import { ActionsColumn } from '@patternfly/react-table';
+
+import useFeatureFlags from 'hooks/useFeatureFlags';
+import { ComplianceScanConfigurationStatus } from 'services/ComplianceScanConfigurationService';
+
+import { scanConfigDetailsPath } from './compliance.scanConfigs.routes';
+
+// Component for scan configs table table page corresponds to ScanConfigActionDropdown for scan config details page.
+
+// Caller is responsible for conditional rendering only if READ_WRITE_ACCESS level for Compliance resource.
+
+export type ScanConfigActionsColumnProps = {
+    handleDeleteScanConfig: (scanConfigResponse: ComplianceScanConfigurationStatus) => void;
+    handleRunScanConfig: (scanConfigResponse: ComplianceScanConfigurationStatus) => void;
+    handleSendReport: (scanConfigResponse: ComplianceScanConfigurationStatus) => void;
+    scanConfigResponse: ComplianceScanConfigurationStatus;
+};
+
+function ScanConfigActionsColumn({
+    handleDeleteScanConfig,
+    handleRunScanConfig,
+    handleSendReport,
+    scanConfigResponse,
+}: ScanConfigActionsColumnProps): ReactElement {
+    const history = useHistory();
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const isComplianceReportingEnabled = isFeatureFlagEnabled('ROX_COMPLIANCE_REPORTING');
+
+    const { id, lastExecutedTime, scanConfig } = scanConfigResponse;
+    const { notifiers } = scanConfig;
+    const scanConfigUrl = generatePath(scanConfigDetailsPath, {
+        scanConfigId: id,
+    });
```

Comment:
Although this is much less likely to cause an error during render than in [global search](https://github.com/stackrox/stackrox/pull/11350/files#r1625973012), due to the indirection and general usage of the latter, I wonder if it would be worthwhile to still handle the potential thrown error in some way?

I also imagine it might be possible to have a wrapper around `generatePath` that not only handles the try/catch, but uses string template literals to catch errors at compile time. 🤔 Definitely an experiment for a later date though.

Reply:
Yes, I liked what you did in global search.